openlibrary/templates/type/edition/view.html

```diff
@@ -79,9 +79,9 @@
         schema_attribute = cond(include_schema, ' itemprop="author"', '')
         authors = cond(work, work and work.get_authors(), edition and edition.get_authors())
         if authors:
-            return ', '.join('<a href="%s"%s>%s</a>' % (a.url(), schema_attribute, truncate(a.name, 40)) for a in authors)
+            return ', '.join('<a href="%s">%s</a>' % (a.url(), schema_attribute, truncate(a.name, 40)) for a in authors)
```

Comment:
There are three variables `a.url(), schema_attribute, truncate(a.name, 40)` so their should also be three `%s`.